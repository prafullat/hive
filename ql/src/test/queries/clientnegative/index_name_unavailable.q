CREATE INDEX srcpart TYPE PROJECTION ON TABLE src(key);
CREATE INDEX srcpart TYPE PROJECTION ON TABLE src(key);