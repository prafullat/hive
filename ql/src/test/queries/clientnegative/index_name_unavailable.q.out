PREHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
PREHOOK: type: CREATEINDEX
POSTHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
POSTHOOK: type: CREATEINDEX
PREHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
PREHOOK: type: CREATEINDEX
FAILED: Error in metadata: org.apache.hadoop.hive.ql.metadata.HiveException: Index srcpart already exists on table src, db=default
FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.exec.DDLTask
PREHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
PREHOOK: type: CREATEINDEX
POSTHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
POSTHOOK: type: CREATEINDEX
PREHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
PREHOOK: type: CREATEINDEX
FAILED: Error in metadata: org.apache.hadoop.hive.ql.metadata.HiveException: Index srcpart already exists on table src, db=default
FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.exec.DDLTask
PREHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
PREHOOK: type: CREATEINDEX
POSTHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
POSTHOOK: type: CREATEINDEX
PREHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
PREHOOK: type: CREATEINDEX
FAILED: Error in metadata: org.apache.hadoop.hive.ql.metadata.HiveException: Index srcpart already exists on table src, db=default
FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.exec.DDLTask
PREHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
PREHOOK: type: CREATEINDEX
POSTHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
POSTHOOK: type: CREATEINDEX
PREHOOK: query: CREATE INDEX srcpart TYPE COMPACT ON TABLE src(key)
PREHOOK: type: CREATEINDEX
FAILED: Error in metadata: org.apache.hadoop.hive.ql.metadata.HiveException: Index srcpart already exists on table src, db=default
FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.exec.DDLTask
